tsconfig.json

```diff
@@ -5,21 +5,9 @@
     "experimentalDecorators": true,
     "target": "ES5",
     "sourceMap": true,
```

Comment:
Why do you use es5 with powerbi utils 2.x.x and api 2.5+, why not es6

Reply:
Planned ES6 migration in the next version